content/influxdb/cloud-dedicated/reference/internals/security.md

```diff
@@ -222,12 +237,78 @@ Users can configure the following security controls:
+The user interacts with the `influxctl` command line tool to manage the workload cluster, including creating [database tokens](#database-tokens) for database read and write access.
+
+#### Management tokens
+
+Management tokens authenticate user accounts to the Granite service and provide authorizations for workload cluster management activities, including:
+ 
+ - account, pricing, and infrastructure management
+ - inviting, listing, and deleting users
+ - creating, listing, and deleting, and databases
+ - creating, listing, and deleting database tokens
+
+Management tokens consist of the following:
+
+- An access token string (sensitive)
+- A permissions set for management activities (configured by InfluxData during initial provisioning)
+- A mandatory 1 hour expiration
+
+When a user issues a command using the `influxctl` command-line tool, `influxctl` sends the management token string with the request to the server, where Granite validates the token (for example, using Auth0).
+If the management token is valid and not expired, the service then compares the token's permissions against the permissions needed to complete the user's request.
+
+Only valid unexpired tokens that have the necessary permissions sets are authorized to perform management functions with InfluxDB Cloud Dedicated.
+Following security best practice, management tokens are never stored on InfluxDB Cloud Dedicated (Granite or workload cluster) servers, which prevents token theft from the server.
+On the client (the user's system), the management token is stored on disk with restricted permissions for `influxctl` to use on subsequent runs.
+For example, a user's Linux system would store the management token at  `~/.cache/influxctl/*.json` with `0600` permissions (that is, owner read and write, and no access for _group_ or _other_).
+
+#### Database tokens
+
+Database tokens provide authorization for users and client applications to read and write data and metadata in an InfluxDB Cloud Dedicated database.
+All data write and query API requests require a valid database token with sufficient permissions.
+_**Note:** an all-access management token can't read or write to a database because it's not a database token._
+
+Database tokens consist of the following:
+
+- An identifier
+- A description
+- A permissions set for reading from a database, writing to a database, or both
+- A base64-encoded 512-bit random string (the API key string)
+
+When a user successfully creates a database token, the InfluxDB Cloud Dedicated Granite server reveals the new database token to the user as an API key string - the key string is only visible when it's created.
+The user is responsible for securely storing and managing the API key string.
+
+Following security best practice, database tokens are never stored on InfluxDB Cloud Dedicated (Granite or workload cluster) servers, which prevents token theft from the server.
```

Comment:
s/database tokens/the raw API key string of database tokens/ (or similar)

We do store the other portions of the database tokens (ie, the identifier, the description, the permissions set), but we do NOT store the raw API key string. I think it would be more clear to say that here as then it sets up the reader for the next sentence.